src/components/main.js

```diff
@@ -5,6 +5,9 @@ export default function Header() {
   const [result, setResult] = useState(null);
   const [file, setFile] = useState(null);
 
+  // Above this value we should consider this a seizure
+  const predThreshold = 70;
```

Comment:
This is a great addition! After we do some more work with our dataset/model we can figure out a data-driven way to set this value as well!